tools/rollup.js

```diff
@@ -71,26 +71,25 @@ if (process.argv.includes('-beta')) {
   };
 
   // user script
-  await bundle.write({
-    ...sharedBundleOpts,
-    banner: metadata + license,
-    // file: '../builds/test/rollupOutput.js',
-    file: resolve(buildDir, `${packageJson.meta.path}${channel}.user.js`),
-  });
+  if (process.argv.includes('-script')) {
+    await bundle.write({
+      ...sharedBundleOpts,
+      banner: metadata + license,
+      // file: '../builds/test/rollupOutput.js',
+      file: resolve(buildDir, `${packageJson.meta.path}${channel}.user.js`),
+    });}
 
   // chrome extension
-  const crxDir = resolve(buildDir, 'crx');
-  await bundle.write({
-    ...sharedBundleOpts,
-    banner: license,
-    file: resolve(crxDir, 'script.js'),
-  });
-
-  await copyFile(resolve(__dirname, '../src/meta/eventPage.js'), resolve(crxDir, 'eventPage.js'));
-
-  writeFile(resolve(crxDir, 'manifest.json'), generateManifestJson(packageJson, version, channel));
-
-  for (const file of ['icon16.png', 'icon48.png', 'icon128.png']) {
-    await copyFile(resolve(__dirname, '../src/meta/', file), resolve(crxDir, file));
-  };
-})();
+  if (process.argv.includes('-crx')) {
+	  const crxDir = resolve(buildDir, 'crx');
+	  await bundle.write({
+		  ...sharedBundleOpts,
+		  banner: license,
+		  file: resolve(crxDir, 'script.js'),
+	  });
+	  await copyFile(resolve(__dirname, '../src/meta/eventPage.js'), resolve(crxDir, 'eventPage.js'));
+	  writeFile(resolve(crxDir, 'manifest.json'), generateManifestJson(packageJson, version, channel));
+	  for (const file of ['icon16.png', 'icon48.png', 'icon128.png']) {
+		  await copyFile(resolve(__dirname, '../src/meta/', file), resolve(crxDir, file));
+	  };}
+  })();
```

Comment:
![Code_lxAdHI8tt5](https://user-images.githubusercontent.com/126348645/223191729-53e8924f-f7c2-40a0-abe2-330f3cfd9478.png)
You're mixing tabs and spaces in the indentation here.
What editor are you using?

Reply:
I use VS Code for this project.
But i'm pretty sure its intentional, its when you add the arument `-crx` it makes the crx folder.
Just tested it and it works as intended. Its so you can choose what to install, either the userscript and or crx

Reply:
I'm talking about the indentation. 
![Code_vnauVYekdl](https://user-images.githubusercontent.com/126348645/223201147-f816410f-a4ac-4cd8-8b7c-e37653073055.png)